pages/Header.js

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
+import Link from 'next/link';
 
 const Header = () => {
   return (
     <header className='nav-header'>
         <nav>
```

Comment:
For consistency reasons stick to spaces: 2 as the rest of the files.

Reply:
Solved. Kindly review again